integration_tests/grpc_rest/pki/helpers.go

```diff
@@ -1292,4 +1391,210 @@ func Demo(suite *utils.TestSuite) {
+	//
+	//	// Add PAI certificate
+	//	msgAddX509Cert = pkitypes.MsgAddX509Cert{
+	//		Cert:   testconstants.PAICertWithPidVid,
+	//		Signer: venAcc65522.Address,
+	//	}
+	//	_, err = suite.BuildAndBroadcastTx([]sdk.Msg{&msgAddX509Cert}, venName65522, venAcc65522)
+	//	require.NoError(suite.T, err)
+	//
+	//	msgAddPkiRevDistPoints = pkitypes.MsgAddPkiRevocationDistributionPoint{
+	//		Signer:               venAcc65522.Address,
+	//		Vid:                  venAcc65522.VendorID,
+	//		IssuerSubjectKeyID:   testconstants.SubjectKeyIDWithoutColons,
+	//		IsPAA:                false,
+	//		Pid:                  8,
+	//		CrlSignerCertificate: testconstants.PAICertWithVid,
+	//		Label:                "label_PAI",
+	//		DataURL:              testconstants.DataURL,
+	//		RevocationType:       1,
+	//	}
+	//	_, err = suite.BuildAndBroadcastTx([]sdk.Msg{&msgAddPkiRevDistPoints}, venName65522, venAcc65522)
+	//	require.NoError(suite.T, err)
+	//
+	//	revDistPoints, _ = GetAllPkiRevocationDistributionPoints(suite)
+	//	require.Equal(suite.T, 2, len(revDistPoints))
+	//
+	//	revDistPoint, _ = GetPkiRevocationDistributionPoint(suite, venAcc65522.VendorID, testconstants.SubjectKeyIDWithoutColons, "label")
+	//	require.Equal(suite.T, venAcc65522.VendorID, revDistPoint.Vid)
+	//	require.Equal(suite.T, "label_PAI", revDistPoint.Label)
+	//	require.Equal(suite.T, testconstants.SubjectKeyIDWithoutColons, revDistPoint.IssuerSubjectKeyID)
+
+	// Update revocation distribution point
+	msgUpdPkiRevDistPoint := pkitypes.MsgUpdatePkiRevocationDistributionPoint{
+		Signer:               vendorAccount.Address,
+		Vid:                  vendorAccount.VendorID,
+		IssuerSubjectKeyID:   testconstants.SubjectKeyIDWithoutColons,
+		CrlSignerCertificate: testconstants.PAACertWithNumericVid,
+		Label:                "label",
+		DataURL:              "https://url2.data.dclmodel",
+	}
+
+	_, err = suite.BuildAndBroadcastTx([]sdk.Msg{&msgUpdPkiRevDistPoint}, vendorName, vendorAccount)
+	require.NoError(suite.T, err)
+
+	revDistPoint, _ = GetPkiRevocationDistributionPoint(suite, vendorAccount.VendorID, testconstants.SubjectKeyIDWithoutColons, "label")
+	require.Equal(suite.T, vendorAccount.VendorID, revDistPoint.Vid)
+	require.Equal(suite.T, "https://url2.data.dclmodel", revDistPoint.DataURL)
+	require.Equal(suite.T, testconstants.SubjectKeyIDWithoutColons, revDistPoint.IssuerSubjectKeyID)
+
+	// Delete revocation distribution point
+	msgDelPkiRevDistPoint := pkitypes.MsgDeletePkiRevocationDistributionPoint{
+		Signer:             vendorAccount.Address,
+		Vid:                vendorAccount.VendorID,
+		IssuerSubjectKeyID: testconstants.SubjectKeyIDWithoutColons,
+		Label:              "label",
+	}
+
+	_, err = suite.BuildAndBroadcastTx([]sdk.Msg{&msgDelPkiRevDistPoint}, vendorName, vendorAccount)
+	require.NoError(suite.T, err)
+
+	_, err = GetPkiRevocationDistributionPoint(suite, vendorAccount.VendorID, testconstants.SubjectKeyIDWithoutColons, "label")
+	suite.AssertNotFound(err)
+
```

Comment:
Please also call  GetPkiRevocationDistributionPointsBySubject (suite.AssertNotFound(err) is expected)